### PR TITLE
fix: APPS-2976 & APPS-3011 updates to cardMeta - BlogDetail & EventDetail pages

### PR DESCRIPTION
**Pull request by jendiamond**
Connected to:
+ [APPS-2976](https://jira.library.ucla.edu/browse/APPS-2976) - Category in CardMeta should link to the series entry on **EventDetail** page
+ [APPS-3011](https://jira.library.ucla.edu/browse/APPS-3011) - Move the about-author field out of CardMeta into a RichText component to fix truncation on the **BlogDetail** page 

---

The **category is now linkable** here:  
https://deploy-preview-61--test-ftva.netlify.app/events/la-r%C3%A9gion-centrale-03-08-24

Look for the **About the Author** here:
https://deploy-preview-61--test-ftva.netlify.app/blog/test-tom-reeds-for-members-only-black-perspectives-on-local-l-a-tv

---

**Notes:**
This is a two part PR. Updates are made in [ucla-library-website-components](https://github.com/UCLALibrary/ucla-library-website-components/pull/639) also.

1. `gql/queries/FTVAEventDetail.gql`
    + Add the uri for the series in the ftvaEventSeries entry
1. `pages/events/[slug].vue`
    + Add a NuxtLink for the category in CardMeta using the named slot `linkedcategoryslot`
1. `pages/blog/[slug].vue`
    + Add a heading for the About the Author field & styles it
1. `package.json & .lock`
    + Update the component library to bring in the changes to the CardMeta component to support this work
    + Update the design tokens to add all the current styles and colors for the FTVA site
1. `nuxt.config.ts`
    + Add the [FTVA design tokens](https://github.com/UCLALibrary/design-tokens/blob/main/scss/_tokens-ftva.scss) to the Nuxt config

**Checklist:**
- [x] I added github label for semantic versioning
- [x] I double checked it looks like the designs
- [x] UX has reviewed this PR

[APPS-2976]: https://uclalibrary.atlassian.net/browse/APPS-2976?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APPS-3011]: https://uclalibrary.atlassian.net/browse/APPS-3011?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pages/blog/[slug].vue [hunk 1]
```diff
@@ -254,9 +268,11 @@ useHead({
     }
   }
 
-  // if there is no author content, hide the 'about the author' heading
-  :deep(.heading-about-author:not(:has(+.rich-text))) {
-    display: none;
+  .about-the-author {
+    @include ftva-subtitle-2;
+    color: #115daf;
+    /* $accent-blue; */
```
Comment (tinuola) [inline pages/blog/[slug].vue:274]: @jendiamond Is the design token not working?

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -7,42 +7,42 @@ settings:
 dependencies:
   '@pinia/nuxt':
     specifier: ^0.5.1
-    version: 0.5.1(typescript@5.4.5)(vue@3.4.31)
+    version: 0.5.5(typescript@5.6.3)(vue@3.5.12)
```
Comment (pghorpade) [inline pnpm-lock.yaml:10]: copy the pnpm lock file in your branch from main. There should not be any change to lock file without the change in package.json

File: pages/blog/[slug].vue [hunk 3]
```diff
@@ -171,7 +170,19 @@ useHead({
...
           </template>
         </CardMeta>
+
+        <h3 class="about-the-author">
+          About the Author
+        </h3>
+
+        <RichText
+          v-if="page?.aboutTheAuthor"
+          data-test="aboutTheAuthor"
```
Comment (tinuola) [inline pages/blog/[slug].vue:180]: @jendiamond What is this `data-test` attribute for?

**Review by pghorpade — CHANGES_REQUESTED** [review]

File: pages/blog/[slug].vue (hunk 3 above)
Comment (jendiamond) [reply pages/blog/[slug].vue:180]: Data-test is for the cypress tests

File: pages/blog/[slug].vue (hunk 1 above)
Comment (jendiamond) [reply pages/blog/[slug].vue:274]: the design token did not work here

File: pnpm-lock.yaml (hunk 2 above)
Comment (jendiamond) [reply pnpm-lock.yaml:10]: I had done thst because it was getting convoluted. I will do it again

**Review by pghorpade — APPROVED** [review]
👍

**Review by pghorpade — APPROVED** [review]
👍

**Pull request merged**